StatBot/Models/DataModel.cs

```diff
@@ -26,7 +27,9 @@ public static List<Statistic> Statistics
             get
             {
                 var db = new PostgresStatsRepository();
-                return db.GetAll().ToList();
+                var nonsorted = db.GetAll().ToList();
+                IEnumerable<Statistic> result = nonsorted.OrderBy(x => x.Id);
+                return result.ToList();
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor 'Statistics' into a method, properties should not copy collections. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://172.21.13.172:9091/coding_rules#rule_key=csharpsquid%3AS2365)